jax/_src/numpy/lax_numpy.py

```diff
@@ -3965,8 +3965,8 @@ def append(
 
   Args:
     arr: original array.
-    values: values to be appended to the array.
-    axis: axis along which to append values.
+    values: values to be appended to the array. The ``values`` must have the same number of dimensions as ``arr``, and all dimensions must match except in the specified axis.
```

Comment:
For docstrings, we should manually break lines after ~80ish characters, otherwise the formatting is very poor when displayed in a terminal. See other docstrings in this file for examples of how to indent this.